clients/src/test/java/org/apache/kafka/common/utils/UtilsTest.java

```diff
@@ -681,4 +683,15 @@ public void testDiff() {
         assertThat(diff, is(mkSet("a", "b")));
         assertThat(diff.getClass(), equalTo(TreeSet.class));
     }
+
+    @Test
+    public void testPropsToMap() {
+        Properties props = new Properties();
+        props.put(1, 2);
+        assertThrows(ConfigException.class, () -> Utils.propsToMap(props));
+        props.remove(1);
+        // the non-string value is legal
+        props.put("key", 22);
+        assertEquals(Utils.propsToMap(props).get("key"), 22);
```

Comment:
Should we assert a string valued parameter too?

Reply:
done